### ISSUE TITLE
CorrEns.plot() doesn't allow to pass ax argument

### ISSUE THREAD

**Issue by khider** [LinkedEarth/Pyleoclim_util#175]
**Describe the bug**
the function should allow to take ax as an argument but fig is defined in the function first. 

**To Reproduce**
```
import matplotlib.pyplot as plt
fig,ax = plt.subplots(2,2)
corr_ens.plot(ax=ax[0,0],mute=True)
```

**Screenshots**
![image](https://user-images.githubusercontent.com/11758571/139491826-9be84442-2a9f-489e-baf5-950ee4459842.png)

**Comment by CommonClimate** [comment]
I thinbk the issue has to do with whether a fig handle has been defined internally or not. Passing an ax argument deactivates the creation of a figure, and therefore the bit at the end has no fig to work with.  I copied over the solution used in `PSD.plot()`:

```
if 'fig' in locals():
            if 'path' in savefig_settings:
                plotting.savefig(fig, settings=savefig_settings)
            else:
                if not mute:
                    plotting.showfig(fig)
            return fig, ax
        else:
            return ax
```

We should check that all Class-specific plotting functions have it: yes for MultipleSeries, Coherence, Scalogram, PSD, MultiplePSD. I think CorrEns was the only class where this was missing.

**Comment by CommonClimate** [comment]
Solved by c86428501b769b51ffbc4f557e6ee01b5b9dd2e9
